### PR TITLE
Fix elasticsearch env variables

### PR DESCRIPTION
**Pull request by simonvadee**

### CODE REVIEW

**Comment by vmttn** [comment]
1. Can you change the default variables in the dot env file ?
https://github.com/arkhn/pyfhirstore/blob/b9a4f84ac71b2d160c9778c38988b7e55e4eba31/.env#L3
2. CI tests are not ok ?
3. Tests are failing more on this branch than on master (49 ko on master, 111 ko on this branch). (followed the steps in the `README.md`)

File: .env [hunk 1]
```diff
@@ -1,3 +1,7 @@
 MONGO_USERNAME=arkhn
 MONGO_PASSWORD=SuperSecurePassword2019
-ES_URL=http://elastic:SuperSecurePassword2019@localhost:9200
\ No newline at end of file
+
+ES_USERNAME=elastic
+ES_PASSWORD=SuperSecurePassword2019>
```
Comment (Jasopaum) [inline .env]: Typo: trailing `>`

File: scripts/initiate_rep_set.sh [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 i=0
 while [ $i -lt 20 ]; do
-  curl -s "${ES_URL}/_cluster/health?wait_for_status=yellow&timeout=10s" > /tmp/output
+  curl -s "http://${ES_USERNAME}:${ES_PASSWORD}@${ES_HOST}:${ES_PORT}/_cluster/health?wait_for_status=yellow&timeout=10s" > /tmp/output
```
Comment (Jasopaum) [inline scripts/initiate_rep_set.sh:11]: Is it ok to handle the case where we don't have a password in the python code but not here?

Comment (simonvadee) [reply scripts/initiate_rep_set.sh:11]: no it's not, done

**Review by Jasopaum — APPROVED** [review]

**Pull request merged**